sui-execution/latest/sui-adapter/src/execution_engine.rs

```diff
@@ -220,100 +220,85 @@ fn execute_transaction<Mode: ExecutionMode>(
         gas_charger.no_charges(),
         "No gas charges must be applied yet"
     );
+
     let is_genesis_tx = matches!(transaction_kind, TransactionKind::Genesis(_));
     let advance_epoch_gas_summary = transaction_kind.get_advance_epoch_tx_gas_summary();
 
     // We must charge object read here during transaction execution, because if this fails
     // we must still ensure an effect is committed and all objects versions incremented
     let result = gas_charger.charge_input_objects(temporary_store);
     let mut result = result.and_then(|()| {
-        let mut execution_result = if deny_cert {
-            Err(ExecutionError::new(
-                ExecutionErrorKind::CertificateDenied,
-                None,
-            ))
-        } else {
-            execution_loop::<Mode>(
+         let mut execution_result = if deny_cert {
+                Err(ExecutionError::new(
+                    ExecutionErrorKind::CertificateDenied,
+                    None
+                ))
+            } else {
+        execution_loop::<Mode>(
```

Comment:
Some weird formatting here. If you comment out the `checked_arithmetic! {` at the top of this file, cargo fmt will work, then you can uncomment